### PR TITLE
RESTWS-716:Orders resource returns no results when only status is specified

### PR DESCRIPTION
**Pull request by gitcliff**
link:https://issues.openmrs.org/browse/RESTWS-716
Orders resource returns no results when only status is specified

### CODE REVIEW

File: omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java [hunk 1]
```diff
@@ -39,11 +40,11 @@
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.OrderResource1_8;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PatientResource1_8;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.List;
+import io.swagger.models.Model;
+import io.swagger.models.ModelImpl;
```
Comment (dkayiwa) [inline omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java]: Are these imports necessary for the ticket at hand?

Comment (dkayiwa) [reply omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java]: Or are they simply formatting changes?

Comment (gitcliff) [reply omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java]: @dkayiwa they are formatting changes

Comment (dkayiwa) [reply omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java]: Can you use a separate pull request for formatting changes?

File: omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java [hunk 2]
```diff
@@ -589,4 +589,12 @@ public void invalidCareCenterShouldThrowException() throws Exception {
 		        );
 		handle(req);
 	}
+	
+	@Test(expected = ObjectNotFoundException.class)
+	public void shouldReturnExceptionIfNoUuidIsSpecified() throws Exception {
+		MockHttpServletRequest req = newGetRequest(getURI(),
+
```
Comment (mozzy11) [inline omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java]: @gitcliff  why this space here ??

File: omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java (hunk 1 above)
Comment (gitcliff) [reply omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java]: @dkayiwa i have resolved the formatting changes

File: omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java (hunk 2 above)
Comment (gitcliff) [reply omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java]: i have fixed it

**Comment by gracepotma** [comment]
@dkayiwa can this be merged?

**Comment by dkayiwa** [comment]
@gitcliff i find it confusing to return an `ObjectNotFoundException`, even when the `patient` parameter is not specified.
My expectation is that a call like `order?status=active` should result into a bad request status code as Darius clearly put it in one of the ticket comments.

**Pull request closed without merging**